### PR TITLE
Ibimina hero component

### LINKED ISSUE #49: Ibimina - Hero Component
The Hero section should include a brief description of the community directory and its purpose. A clear CTA(Call to Action) should be included, inviting users to either create their own listing

### PR DESCRIPTION
**Pull request by ibimina**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other (please specify here: Component Creation)
  
  
## Description of what this Pull Request does
Hero Component
The Component should include a brief description of the community directory and its purpose. A clear CTA(Call to Action) should be included on the banner, inviting users to either create their own listing or explore the listings already available.

## Related Tickets & Documents
- Closes #49 
- Related Issues: #13 
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your  PR. Feel free to use screenshots and/or short videos)
### Hero Component Screenshot
FireShot Capture 007 - Create Next App - localhost
  ### Landing page Screenshot
FireShot Capture 013 - Create Next App - localhost

## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [x] No, and here is why: __explain here__

### CODE REVIEW

File: src/styles/Home.module.css [hunk 1]
```diff
@@ -1,278 +0,0 @@
-.main {
```
Comment (msarit) [inline src/styles/Home.module.css:1]: @ibimina Seeing as this file has nothing in it (as I saw locally) why are you still including it in this PR?

File: src/pages/index.js [hunk 2]
```diff
@@ -15,109 +12,8 @@ export default function Home() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <main className={styles.main}>
```
Comment (msarit) [inline src/pages/index.js:14]: @ibimina There are no styles defined in `Home.modules.css` so what are you importing here? 🤔

**Review by msarit — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 2 above)
Comment (ibimina) [reply src/pages/index.js]: I have removed the empty stylesheet import

File: src/styles/Home.module.css (hunk 1 above)
Comment (ibimina) [reply src/styles/Home.module.css:1]: I have deleted the empty stylesheet

File: src/styles/globals.css [hunk 3]
```diff
@@ -1,107 +1,18 @@
...
-  color: inherit;
   text-decoration: none;
-}
-
-@media (prefers-color-scheme: dark) {
-  html {
-    color-scheme: dark;
-  }
-}
+}
```
Comment (jjleigh) [inline src/styles/globals.css:18]: Add a new line here.

Comment (ibimina) [reply src/styles/globals.css]: done

**Review by msarit — APPROVED** [review]
@ibimina Resolve conflicts then this PR should be good to go! 🚀

File: src/pages/index.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ export default function Home() {
...
+        <Hero />
         <section className={styles.listings_container}>
```
Comment (msarit) [inline src/pages/index.js:19]: @ibimina When I pulled your branch down and ran the app, I do not see this `<section>` code in my local copy.

Please update this branch with latest `main` thanks!

**Review by msarit — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 4 above)
Comment (ibimina) [reply src/pages/index.js:19]: okay

File: src/components/landing/hero.module.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
+.hero_section{
```
Comment (msarit) [inline src/components/landing/hero.module.css:1]: @ibimina We have decided to use `camelCase`

**Review by msarit — CHANGES_REQUESTED** [review]

File: src/components/landing/hero.module.css (hunk 5 above)
Comment (ibimina) [reply src/components/landing/hero.module.css:1]: ok

File: src/components/landing/Hero.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import styles from "./hero.module.css";
+function Hero() {
+  return (
+    <section className={styles.heroSection}>
+      <div className={styles.heroWrap}>
+        <h1>find your people</h1>
```
[the remark this answers is not in the document]
Comment (ibimina) [reply src/components/landing/Hero.js]: "find|" casing updated

**Review by msarit — APPROVED** [review]

**Pull request merged**